apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -623,14 +623,23 @@ type QueryLimit struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:number",displayName="Max Tags per User"
 	MaxBytesPerTagValues *int `json:"maxBytesPerTagValues,omitempty"`
-	// MaxSearchBytesPerTrace defines the maximum size of search data for a single
+
+	// DEPRECATED. MaxSearchBytesPerTrace defines the maximum size of search data for a single
 	// trace in bytes.
 	// default: `0` to disable.
 	//
 	// +optional
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:number",displayName="Max Traces per User"
-	MaxSearchBytesPerTrace *int `json:"maxSearchBytesPerTrace"`
+	MaxSearchBytesPerTrace *int `json:"maxSearchBytesPerTrace,omitempty"`
+
+	// MaxSearchDuration defines the maximum search duration.
```

Comment:
Duration of what? The query execution in the Tempo engine or duration of a span/trace? 

Maybe something we should clarify?

Reply:
You're right, that comment wasn't very clear. I've updated it.